WordPress/Classes/ViewRelated/Site Creation/Web Address/WebAddressWizardContent.swift

```diff
@@ -35,7 +36,10 @@ final class WebAddressWizardContent: CollapsableHeaderViewController {
     private let table: UITableView
     private let searchHeader: UIView
     private let searchTextField: SearchTextField
+    private let searchBar = UISearchBar()
     private var sitePromptView: SitePromptView!
+    private let siteCreationEmptyTemplate = SiteCreationEmptySiteTemplate()
+    private lazy var hostingController = UIHostingController(rootView: siteCreationEmptyTemplate)
```

Comment:
nit: `hostingController` naming is a bit generic, would be great to rename it to something related to `siteCreationEmptyTemplate` view. So the reader knows that the `hostingController` is for the `Empty Site` view.